backend/migrating/v2/migrate-to-v2.sh

```diff
@@ -37,11 +34,16 @@ echo -e "${blue_text}Configuring environment variables for the migration${defaul
 cd $manage_script_dir
 activate_venv
 
+# TODO: Update to support running from container instead of setting these envs
+# VERSION=test docker compose -f docker-compose.yaml run backend .venv/bin/python manage.py create_schema
 echo -e "${blue_text}Running schema creation command...${default_text}"
-python manage.py create_v2_schema || { echo "Schema creation failed"; exit 1; }
+DB_HOST="localhost" REDIS_HOST="localhost" CELERY_BROKER_URL="redis://localhost:6379" \
+python manage.py create_schema || { echo "Schema creation failed"; exit 1; }
 
 echo -e "${blue_text}Running schema migration command...${default_text}"
+DB_HOST="localhost" REDIS_HOST="localhost" CELERY_BROKER_URL="redis://localhost:6379" \
```

Comment:
@chandrasekharan-zipstack Also ideally only `DB_HOST` is required, as we don't need Redis or Celery for any db migrations.

Reply:
@hari-kuriakose the need for Redis comes from how we register the celery tasks - didn't want to mess / check it in the scope of this PR so ended up including it. More in [this comment](https://github.com/Zipstack/unstract/pull/820#discussion_r1817771020).

If we adopt `docker compose run` commands - this would change anyway

Reply:
@chandrasekharan-zipstack As observed, direct invocation like this is easier than docker compose run sanity checks and usage requirements.